### DISCUSSION TITLE
Auto-detect new bundles

### DISCUSSION THREAD

**Discussion by devsheva** [shakacode/react_on_rails#1659] [Ideas]
A great feature would be support for auto detection of recent added bundles, so manifest gets reloaded and no app restart is needed. Is this possible?

**Comment by justin808** [comment]
If you want a submit a PR, that would be great.

**Reply by devsheva** [reply]
I'd like to do it for sure, but it will take some time cause I need how the gem really works and how to achieve this behavior
